syncopy/datatype/continuous_data.py

```diff
@@ -420,6 +422,43 @@ def multipanelplot(self, **show_kwargs):
         figax = mp_plotting.plot_AnalogData(self, **show_kwargs)
         return figax
 
+    def save_nwb(self, outpath, nwbfile=None, with_trialdefinition=True, is_raw=True):
```

Comment:
if `nwbfile` is not `None` but points to an existing nwb file, saving did not work. Probably what you mean is starting with an empty `nwbfile` container? However this is not tested..

Reply:
This is not fixed and there is a unit test for it, but see my detailed comment on this topic below.